src/weights.jl

```diff
@@ -250,6 +250,76 @@ eweights(t::AbstractVector, r::AbstractRange, λ::Real) =
 
 # NOTE: No variance correction is implemented for exponential weights
 
+struct UnitWeights{S<:Real, T<:Real} <: AbstractWeights{S, T, V where V<:Vector{T}}
+    sum::S
+end
+
+UnitWeights{T}(s::S) where {S, T} = UnitWeights{S, T}(s)
+
+@doc """
+    UnitWeights{T}(s)
+
+Construct a `UnitWeights` vector with length `s` and weight elements of type `T`.
+All weight elements are identically one.
+""" UnitWeights
+
+eltype(wv::UnitWeights{S, T})    where {S, T} = T
+values(wv::UnitWeights{S, T})    where {S, T} = fill(one(T), length(wv))
+isempty(wv::UnitWeights{S, T})   where {S, T} = iszero(wv.sum)
+length(wv::UnitWeights{S, T})    where {S, T} = Int(wv.sum)
+Base.size(wv::UnitWeights{S, T}) where {S, T} = Tuple(length(wv))
```

Comment:
Import `size` for consistency.

Reply:
`size` was not imported, though we'd already extended it in line 27. I will explicitly import it in the next pull request, when we simplify the weight types.

Reply:
Better do this now, commits should make sense in isolation.